chapters/planning-data-work.tex

```diff
@@ -517,21 +497,23 @@ \subsection{Documenting and organizing code}
 Even you will probably not remember the exact choices that were made in a couple of weeks.
 Therefore, you must document your precise processes in your code.
 
-Code organization means keeping each piece of code in an easily findable location.
+Code organization means keeping each piece of code in an easy-to-find location.
   \index{code organization}
 Breaking your code into independently readable ``chunks'' is one good practice on code organization.
 You should write each functional element as a chunk that can run completely on its own,
 to ensure that each component does not depend on a complex program state
 created by other code chunks that are not obvious from the immediate context.
 One way to do this is to create sections where a specific task is completed.
 So, for example, if you want to find the line in your code where a variable was created,
-you can go straight to \texttt{PART 2: Create new variables},
+you can go straight to \texttt{PART 2: Prepare folder paths and define programs},
 instead of reading line by line through the entire code.
 RStudio, for example, makes it very easy to create sections,
 and it compiles them into an interactive script index for you.
 In Stata, you can use comments to create section headers,
-though they're just there to make the reading easier and don't have functionality.
-You should also add an index in the code header by copying and pasting section titles.
+though they're just there to make the reading easier and don't have functionality;
+refer to line 24 of the example .do file.
+You should also add an index in the code header by copying and pasting section titles;
+refer to lines 8-10 in the example .do file.
```

Comment:
Agree on how to write do-file

Reply:
ok updated all to do-file in 7a73864f826d0a9059d2f3240118a1023dfd78c8